Documentation/ApiOverview/Backend/BackendModules/_BackendModuleController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use TYPO3\CMS\Backend\Template\ModuleTemplate;
+use TYPO3\CMS\Backend\Template\ModuleTemplateFactory;
+use TYPO3\CMS\Extbase\Mvc\Controller\ActionController;
+
+final class BackendModuleController extends ActionController
+{
+    public function __construct(
+        private readonly ModuleTemplateFactory $moduleTemplateFactory,
+        // ..
+    ) {}
+
+    protected function initializeModuleTemplate(ServerRequestInterface $request): ModuleTemplate
```

Comment:
Why not override initializeAction() method? This is always called before an action. So no need to call initialiteModuleTemplate() manually in each action - which is error-prone (can be left out unintended). Just set $this->moduleTemplate and the use it in the action.